androgee/androgee.py

```diff
@@ -1,74 +1,51 @@
-import os
-import sys
 import random
 import logging
 import discord
-from discord.ext import commands
+from discord.ext import typed_commands as commands
 from pathlib import (
     Path,
-)  # This makes working with files and searching through them a piece of cake
+)
+from androgee.cogs.spam_badwords import BadWords
+from androgee.init import COMMAND_PREFIX, BOT_TOKEN, MOD_ROLE_ID
 
-# I think it'd be better if we check if environment
-# variables are present before doing anything else.
-try:
-    mod_role_id = os.environ["mod_role_id"]
-    mod_role_name = os.environ["mod_role_name"]
-    COMMAND_PREFIX = os.environ["DISCORD_PREFIX"]
-    BOT_TOKEN = os.environ["DISCORD_TOKEN"]
-except KeyError as e:
-    print(f"The {e} environment variable is missing! Androgee cannot run!")
-    sys.exit(1)
 
 logging.basicConfig(level=logging.WARNING)
-bot = commands.Bot(command_prefix=COMMAND_PREFIX)
+bot: commands.Bot = commands.Bot(command_prefix=COMMAND_PREFIX)
 media_folder = Path(
     __file__, "../media"
 ).resolve()  # Root of the media folder from file.
 
 
-@bot.event
-async def on_ready():
-    print(f"We is logged in as {bot.user}")
+# Super quick and easy random selection, partial credit to citrusMarmelade
+def get_image(ctx):
+    images = list(media_folder.glob(f"{ctx.command}/*"))
```

Comment:
Let's add an issue to deprecate this function and replace it with calls to api like cat api, chuck norris quote api etc.

It's a much better pattern that storing and pulling images from the file system

Reply:
sure